src/ichmod.cpp

```diff
@@ -144,45 +144,57 @@ main( int argc, char **argv ) {
 void
 usage() {
     char *msgs[] = {
-        "Usage: ichmod [-rhvVM] null|read|write|own userOrGroup dataObj|Collection ...",
+        "Usage: ichmod [-rhvVM] AccessLevel userOrGroup dataObject|Collection ...",
```

Comment:
We should change the placeholders to be in all caps and then use those throughout the help text, just like other Unix commands. We do that for `itouch` today.
- `AccessLevel` -> `ACCESS_LEVEL`
- `userOrGroup` -> `USER_OR_GROUP`
- `dataObject`/`Collection` -> `LOGICAL_PATH`

Reply:
ah, will do.